### PR TITLE
Fix for #39

### PR DESCRIPTION
**Pull request by scicco**
This should fix undefined method `migrate' for ActiveRecord::Migrator:Class #39 issue

### CODE REVIEW

**Comment by scicco** [comment]
Hello @natebird I've removed the rc1 version. Thank you for your efforts for this gem.

File: test/test_helper.rb [hunk 1]
```diff
@@ -13,7 +13,12 @@
 
 ActiveRecord::Migration.verbose = false
 ActiveRecord::Base.logger = Logger.new(nil)
-ActiveRecord::Migrator.migrate(File.expand_path('../dummy/db/migrate', __FILE__))
+if Rails.version == '5.2.0'
```
Comment (olbrich) [inline test/test_helper.rb]: What happens with Rails 5.2.1?  This change probably needs to happen for all versions >= 5.2.0

File: test/test_helper.rb [hunk 2]
```diff
@@ -13,7 +13,12 @@
...
+  mc = ActiveRecord::MigrationContext.new(File.expand_path('../dummy/db/migrate', __FILE__))
```
Comment (olbrich) [inline test/test_helper.rb]: not really using `mc`.  Just write it as
```
ActiveRecord::MigrationContext.new(File.expand_path('../dummy/db/migrate', __FILE__)).migrate
```

File: test/test_helper.rb (hunk 1 above)
Comment (scicco) [reply test/test_helper.rb]: Hello @olbrich you're right, I'll commit the fix.

File: test/test_helper.rb (hunk 2 above)
Comment (scicco) [reply test/test_helper.rb]: fixed in latest commit

**Comment by scicco** [comment]
Hello @olbrich is the last change ok for you?

File: test/test_helper.rb [hunk 3]
```diff
@@ -13,7 +13,11 @@
 
 ActiveRecord::Migration.verbose = false
 ActiveRecord::Base.logger = Logger.new(nil)
-ActiveRecord::Migrator.migrate(File.expand_path('../dummy/db/migrate', __FILE__))
+if Rails.version >= '5.2.0'
```
[the remark this answers is not in the document]
Comment (scicco) [reply test/test_helper.rb]: fixed, thank you for your review.

**Comment by olbrich** [comment]
@scicco Looks like you will need to update the branch on master once #42 is merged in order for tests to pass for ruby 2.5

**Pull request merged**